File: spray-calculator/WebApi/Agrovision.Spray_Calculator.Api/Models/FieldModel.cs
using System;

namespace Agrovision.Spray_Calculator.Api.Models
{
    public class FieldModel
    {
        public long Id { get; set; }
        public Guid FieldKey { get; set; }
        public Guid UserKey { get; set; }
        public string Description { get; set; }
        public double FieldSize { get; set; }
        public bool IsActive { get; set; }
    }
}

File: spray-calculator/WebApp/Agrovision/src/app/models/field/field.model.ts
export class FieldModel {

    constructor(description: string, fieldSize: number) {
        this.description = description;
        this.fieldSize = fieldSize;
    }
    id: bigint;
    fieldKey: string;
    userKey: string;
    description: string;
    fieldSize: number;
    isActive: boolean;
}
